sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -112,6 +112,15 @@ private[spark] object HiveUtils extends Logging {
     .booleanConf
     .createWithDefault(true)
 
+  val CONVERT_INSERTING_PARTITIONED_TABLE =
+    buildConf("spark.sql.hive.convertInsertingPartitionedTable")
+      .doc("When set to true, and \"spark.sql.hive.convertMetastoreParquet\" or " +
+        "\"spark.sql.hive.convertMetastoreOrc\" is true, the built-in ORC/Parquet writer is used" +
```

Comment:
Usually we write like \`spark.sql.hive.convertMetastoreOrc\`?

Reply:
I referred `CONVERT_METASTORE_PARQUET_WITH_SCHEMA_MERGING`, will fix it.